src/stores/Binding/Store.ts

```diff
@@ -866,6 +870,22 @@ const getInitialState = (
         );
     },
 
+    setBindingOnIncompatibleSchemaChange: (value) => {
+        set(
+            produce((state: BindingState) => {
+                const bindingUUID = state.currentBinding?.uuid;
```

Comment:
We should pass in this value. Otherwise there is a _tiiiiny_ chance we call this function for one uuid and by the time this actually runs it is another uuid.